lib/MusicBrainz/Script/JSONDump.pm

```diff
@@ -222,7 +222,10 @@ sub _fetch_entities_json {
     my $query;
     my %found;
     my @missing;
-    my ($force_update) = grep { $_ eq $entity_type } @{ $options{force_update} // [] };
+    # XXX Until MBS-10911 can be resolved, have the full dumps fetch
+    # all JSON anew to prevent it from becoming stale for too long.
+    #my ($force_update) = grep { $_ eq $entity_type } @{ $options{force_update} // [] };
+    my $force_update = $options{is_full_dump};
```

Comment:
Would it make sense to still make it possible to force_update on request with options?

Reply:
Good question. It probably wouldn't be that useful, since we don't use `--force-update` for incremental dumps at all and probably won't ever. (This patch already makes it a no-op for full dumps.) This is kind of an esoteric option that'll be removed entirely when the JSON dumps are reworked, and this stuff is only used in our production setup, so I'm fine with a temporary hack here.